test/trivial/deitz/coerce-assign/plusassign_error.good

```diff
@@ -1 +1,8 @@
-plusassign_error.chpl:3: error: Cannot assign to int(32) from int(64)
+plusassign_error.chpl:3: error: unresolved call '+=(int(32), int(64))'
+$CHPL_HOME/modules/internal/ChapelBase.chpl:1873: note: this candidate did not match: +=(lhs: int(8), rhs: int(8))
```

Comment:
Independent of this PR: It's too bad that this is the overload our compiler decides to explain since there's nothing about it that seems like a particularly good match.

That said, the errors for these unsupported += cases are much nicer with this PR.